publication-commons/src/main/java/no/unit/nva/publication/service/impl/ReadResourceService.java

```diff
@@ -98,6 +102,19 @@ public Optional<Resource> getResourceByIdentifier(SortableIdentifier identifier)
         return resource;
     }
 
+    public Stream<TicketEntry> fetchAllTicketsForResource(Resource resource) {
+        var dao = (ResourceDao) resource.toDao();
+        return dao.fetchAllTickets(client)
+                   .stream()
+                   .map(TicketDao::getData)
+                   .map(TicketEntry.class::cast)
+                   .filter(ReadResourceService::isNotRemoved);
+    }
+
+    private static boolean isNotRemoved(TicketEntry ticket) {
+        return !TicketStatus.REMOVED.equals(ticket.getStatus());
+    }
```

Comment:
Moving this method here from resourceService, so I can call in UpdateResourceService.